specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```
@@ -0,0 +1,3590 @@
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "WorkspaceNameParameter": {
+      "name": "workspaceName",
+      "description": "Name of Azure Machine Learning workspace.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "ComputeNameParameter": {
+      "name": "computeName",
+      "description": "Name of the Azure Machine Learning compute.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "AsyncOperationIdParameter": {
+      "name": "asyncOperationId",
+      "description": "Identifier of an asynchronous Azure Machine Learning compute operation.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "LocationParameter": {
+      "name": "location",
+      "description": "The name of the Azure location/region.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "APIVersionParameter": {
+      "in": "query",
+      "name": "api-version",
+      "type": "string",
+      "description": "Version of Azure Machine Learning resource provider API.",
+      "required": true
+    },
+    "PaginationParameter": {
+      "in": "query",
+      "name": "$skiptoken",
+      "type": "string",
+      "description": "Continuation token for pagination.",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "ComputeDefinitionParameter": {
+      "in": "body",
+      "name": "parameters",
+      "description": "Payload with Machine Learning compute definition.",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/ComputeResource"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "ComputeTypeParameter": {
+      "in": "query",
+      "name": "compute-type",
+      "type": "string",
+      "description": "Type of compute to filter by.",
+      "required": false,
+      "x-ms-parameter-location": "method"
+    },
+    "UnderlyingResourceActionParameter": {
+      "in": "query",
+      "name": "underlyingResourceAction",
+      "type": "string",
+      "description": "Delete the underlying compute if 'Delete', or detach the underlying compute from workspace if 'Detach'.",
+      "required": true,
+      "enum": [
+        "Delete",
+        "Detach"
+      ],
+      "x-ms-enum": {
+        "name": "UnderlyingResourceAction",
+        "modelAsString": true
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "PrivateEndpointConnectionName": {
+      "name": "privateEndpointConnectionName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the private endpoint connection associated with the workspace",
+      "x-ms-parameter-location": "method"
+    },
+    "LinkName": {
+      "name": "name",
```

Could we please change this to `linkName`? Since there are a lot of URI referencing this but with the name of `linkName`.
```suggestion
      "name": "linkName",
```